src/App.tsx

```diff
@@ -15,47 +15,86 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+enum SortType {
+  NONE,
+  ALPHABET,
+  LENGTH,
```

Comment:
```suggestion
  None,
  Alphabetically,
  ByLength,
```

Reply:
uppercase values refers to some global variables like
```
const NUMBERS_IN_WEEK = 7;
```
for enum, it's better to use capitalized ones